tests/integration/conftest.py

```diff
@@ -3,6 +3,8 @@
 
 """Fixtures for Jenkins-k8s-operator charm integration tests."""
 
+# pylint: disable=too-many-lines
+
```

Comment:
Perhaps we should consider splitting the conftest or move the fixtures to according modules - if they're only used in one specific test.

Reply:
Good point, I started with moving the auth_proxy related fixtures to the test module and removed this ignore line. I'll refactor more in a follow-up PR.